AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
```

Comment:
Looks like this isn't stable yet, right? Once it is we can hopefully just rely on that entirely for package tests.